src/main/plugins/org.dita.pdf2/xsl/fo/loc.xsl

```diff
@@ -0,0 +1,231 @@
+<?xml version='1.0'?>
+
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:fo="http://www.w3.org/1999/XSL/Format"
+                xmlns:dita-ot="http://dita-ot.sourceforge.net/ns/201007/dita-ot"
+                xmlns:dita2xslfo="http://dita-ot.sourceforge.net/ns/200910/dita2xslfo"
+                xmlns:opentopic="http://www.idiominc.com/opentopic"
+                xmlns:opentopic-index="http://www.idiominc.com/opentopic/index"
+                xmlns:ot-placeholder="http://suite-sol.com/namespaces/ot-placeholder"
+                exclude-result-prefixes="opentopic opentopic-index dita2xslfo ot-placeholder xs"
+                version="2.0">
+
+  <!-- The variable changeset contains a list of all change-item elements. -->
+  <xsl:variable name="changeset" select="//*[contains (@class, ' relmgmt-d/change-item ')]"/>
+
+  <!-- The variable topicsWithChangeItem contains a list of all topics containing change-item elements. -->
+  <xsl:variable name="topicsWithChangeItem" select="//*[contains (@class, ' topic/topic ')][.//*[contains (@class, ' relmgmt-d/change-item ')]]"/>
+
+  <!-- Keep date format in a variable -->
+  <xsl:variable name="dateFormat">[MNn] [D], [Y]</xsl:variable>

```

Comment:
Date format should be moved to common string variable as it's higly language/locale dependent.